config/plugins/visualizations/rerunio/config/rerunio.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE visualization SYSTEM "../../visualization.dtd">
+<visualization name="Rerun.io" embeddable="true">
+    <description>Spatial and Embodied AI</description>
+    <data_sources>
+        <data_source>
+            <model_class>HistoryDatasetAssociation</model_class>
+            <test test_attr="ext">rrd</test>
```

Comment:
Do you need to add something like https://github.com/guerler/galaxy/blob/rerun_io/config/plugins/visualizations/vizarr/config/vizarr.xml#L17 to make it work for deferred URIs?

Reply:
I am not entirely sure, but this should be tested separately.